types/underscore/index.d.ts

```diff
@@ -5239,6 +5297,12 @@ declare namespace _ {
          */
         noop(): void;
 
+        /**
+         * Wrapped type `any`.
+         * @see _.toPath
+         */
+        toPath(path:any): Array<any>;
```

Comment:
Why did you add this to the `Underscore` and `_Chain` interfaces but not to `UnderscoreStatic`?

Reply:
I will report an error when using it on my end, and it has been corrected so far

Reply:
Modified

Reply:
Modified